File: lib-util/src/main/java/code/util/ImageUtil.java
package code.util;

import android.content.Intent;
import android.net.Uri;

import java.io.File;

public class ImageUtil {
    public static final String TAG = ImageUtil.class.getSimpleName();

    public static void triggerScanning(String imagePath) {
        triggerScanning(new File(imagePath));
    }

    public static void triggerScanning(File image) {
        Uri uri;
        try {
            uri = Uri.fromFile(image);
        } catch (Exception e) {
            //ObfuscationStub8.inject();
            return;
        }
        AppGlobal.getApplication().sendBroadcast(
                new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE, uri));
    }
}

File: lib-util/src/main/java/code/util/ImageUtil.kt
package code.util

import android.content.Intent
import android.net.Uri
import java.io.File

object ImageUtil {
   private val TAG = ImageUtil::class.java.simpleName
    fun triggerScanning(imagePath: String) {
        triggerScanning(File(imagePath))
    }

    
    fun triggerScanning(image: File?) {
        val uri: Uri = try {
            Uri.fromFile(image)
        } catch (e: Exception) {
            //ObfuscationStub8.inject();
            return
        }
        AppGlobal.application!!.sendBroadcast(
            Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE, uri)
        )
    }
}